plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/MultipathSCSIPool.java

```diff
@@ -203,6 +204,11 @@ public Map<String, String> getDetails() {
         return this.details;
     }
 
+    @Override
+    public String getLocalPathFor(String relativePath) {
```

Comment:
no need to override with the same implementation, better add default implementation in the interface